### ISSUE TITLE
empty string launch warning

### ISSUE THREAD

**Issue by ektarum** [ComPHPPuebla/dbal-fixtures#3]
Hi,

When you add empty string for field (not null but empty), ForeignKeyProcessor->isReference launch warning because the code take ```$value[0]``` of an empty string.

yml sample :
```yml
fixtures:
    fixtures_1:
        id: 1
        name: 'fix_name'
        additional: ''
```

**Comment by MontealegreLuis** [comment]
Thank you @ektarum for fixing this one. I've released [3.1.1](https://github.com/ComPHPPuebla/dbal-fixtures/releases/tag/3.1.1) with this fix.

**Comment by ektarum** [comment]
Thx for reactivity and released !
